fastparquet/api.py

```diff
@@ -372,11 +373,11 @@ def remove_row_groups(self, rgs, write_fmd:bool = True,
                 remove_with = default_remove
         if not isinstance(rgs, list):
             rgs = [rgs]
-        rgs_to_remove = row_groups_map(rgs)
```

Comment:
please revert this rename to make my life easier

Reply:
done